amulet/deployer.py

```diff
@@ -162,7 +162,7 @@ def add(self, service_name,
 
         return service
 
-    def add_unit(self, service, units=1, target=None):
+    def add_unit(self, service, units=1, target=None, timeout=300):
```

Comment:
Because I'm dumb and forgot to remove it when I decided against keeping an implicit wait in add_unit.